lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -143,10 +149,10 @@ static std::string getSubModuleName(Operation *oldOp) {
 /// circuit is pure combinational logic. If graph cycle exists, at least one
 /// buffer is required to be inserted for breaking the cycle, which will be
 /// supported in the next patch.
-static FModuleOp createTopModuleOp(handshake::FuncOp funcOp,
+static FModuleOp createTopModuleOp(handshake::FuncOp funcOp, bool hasClock,
```

Comment:
Why bother to make this optional? I think we can just always generate this.  Alternatively, there should be a list of clocks to support multi-clock designs.

Reply:
Good point. Let me try to support multiple clocks.